moondeck/test/test_host.py

```
@@ -0,0 +1,70 @@
+
+ROOT = Path(__file__).resolve().parents[2]
+
+# Named here rather than in a requirements file: each is a test's own dependency (pyserial for the
+# flash tests, wled for the /json shim, markdown for the docs checks), and `uv run --with` is how
+# every other script in MoonDeck reaches one.
+PY_DEPS = ("pytest", "pyserial", "markdown", "wled")
+
+
+def run(cmd, label):
+    print(f"\n=== {label} ===", flush=True)
+    r = subprocess.run(cmd, cwd=ROOT)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)